projects/index.html

```diff
@@ -192,15 +308,15 @@ <h2 class="fh5co-lead">Mentors</h2>
 					</div>
 					<div class="col-md-8 col-md-offset-2 text-center fh5co-section-heading work-box">
 				       <div class="mentor">
-                          <div><img width="260" height="250" src="img/Picture2.jpg" alt="Prof. Ho Lok Sang"><h3><a href="https://zh.wikipedia.org/wiki/何濼生" target="_blank">Prof. Ho Lok Sang</a> @ Chu Hai College of Higher Education</h3></div>
-                          <div><img width="256" height="256" src="img/Picture3.jpg" alt="Germaine Lau - Savantas Policy Institute"><h3><a href="https://www.linkedin.com/in/lau-germaine-761472a/" target="_blank">Germaine Lau</a> @ Savantas Policy Institute</h3></div>
-                          <div><img width="256" height="256" src="img/Picture4.jpg" alt="Jeff Boda - International New York Times / Hop Leaf"> <h3><a href="https://hopleaf.hk" target="_blank">Jeff Boda</a> @ International New York Times / Hop Leaf</h3></div>
-				          <div><img width="256" height="256" src="img/Picture5.jpg" alt="Rubio Chan - GloTravel / Eastern Vision"> <h3><a href="https://www.easternvision.hk/about/" target="_blank">Rubio Chan</a> @ GloTravel / Eastern Vision</h3></div>
-				          <div><img width="256" height="256" src="img/Picture6.jpg" alt="Yue Qiu - Bloomberg"><h3><a href="https://www.linkedin.com/in/yueqiu1/" target="_blank">Yue Qiu</a> @ Bloomberg</h3></div>
-				          <div><img width="256" height="256" src="img/Picture8.jpg" alt="Alex Koo - HK01"><h3><a href="#" target="_blank">Alex Koo</a> @ HK01</h3></div>
-						  <div><img width="256" height="256" src="img/Picture9.jpg" alt="Sammy Fung - Open Source Hong Kong (OSHK)"><h3><a href="https://sammy.hk/" target="_blank">Sammy Fung</a> @ Open Source Hong Kong (OSHK)</h3></div>
-						  <div><img width="256" height="256" src="img/Picture7.jpg" alt="John Siu - HK01"><h3><a href="https://www.hk01.com/search/%E8%95%AD%E8%BC%9D%E6%B5%A9" target="_blank">John Siu</a> @ HK01</h3></div>
-						  <div><img width="256" height="256" src="img/Picture1.jpg" alt="Robin Ewing"><h3><a href="http://robinewing.com" target="_blank">Robin Ewing</a> @ Department of Journalism, HKBU</h3></div>
+                          <div><img width="275" height="275" src="img/Picture2.jpg" alt="Prof. Ho Lok Sang"><h3><a href="https://zh.wikipedia.org/wiki/何濼生" target="_blank">Prof. Ho Lok Sang</a></h3> <p>Chu Hai College of Higher Education</p></div>
```

Comment:
It's better to remove duplicate specification of the image sizes. You can remove those `width="275" height="275"` attributes, assign the `img` a class and use CSS to style them in a unified way.